### PR TITLE
GitHub and markdown #685

### PR DESCRIPTION
**Pull request by praneetharra**
I added one line which I thinks solves the confusion.

### CODE REVIEW

File: pages/githubandmarkdown.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ After you rename it, make sure to post a link to the github.io website you've ju
 
 ![Example of github.io page](uploads/images/examplegithubio.png)
 
+Don't worry if you see a **404** when you access the link, you will set that up in *Step 5*.
```
Comment (mappuji) [inline pages/githubandmarkdown.md]: 404 didn't come from step 5, but it is due to Github needs time to set up our Github page.

CMIIW 😄

**Comment by dogi** [comment]
@praneetharra I kinda agree with @mappuji - cause based on my experience github.io is able to work as fast as in 2min I would say that average is 5min,

I dont remember exactly but last week somewhere I read a message about github.io having delay with new pages ...
... that is why I believe you with a day delay ;)
I am sorry :P

Anyhow bigger problem is that github.io after the fork survives only one renaming
in other words if you first round rename wrong 2nd time renaming github.io page will not be created ...

only solution is to delete the fork and do it right - I mean the first time after the fork

Long story short

Wonder mostly how we can condense above information instead and that without confusing people which your comment about 404 message is doing ;)

Maybe we just add a link to this converation ...

@mappuji @waryhermit @paulbert your feedback is welcome :)

**Comment by praneetharra** [comment]
@dogi Yea, I and @mappuji had a discussion on it today. It is taking time for the pages to activate and I experimented with cloning during the same time earlier. That misled me. I guess we just need to add to the users that it may take time for the page to activate and not to panic if they see a **404**.

**Comment by praneetharra** [comment]
@dogi @mappuji Have a look at the new edit and let me know if any change is required.

**Comment by praneetharra** [comment]
@mappuji can you merge the pull request.

**Pull request merged**